packages/web/src/components/gcds-link/gcds-link.tsx

```diff
@@ -0,0 +1,221 @@
+   */
+  @Prop() download: string | undefined;
+
+  /**
+   * The type specifies the media type of the linked document
+   */
+  @Prop() type: string | undefined;
+
+  /**
+   * Set additional HTML attributes not available in component properties
+   */
+  @State() inheritedAttributes: Object = {};
+
+  /**
+   * Language of rendered component
+   */
+  @State() lang: string;
+
+  /**
+   * Events
+   */
+
+  /**
+   * Emitted when the link has focus.
+   */
+  @Event() gcdsFocus!: EventEmitter<void>;
+
+  /**
+   * Emitted when the link loses focus.
+   */
+  @Event() gcdsBlur!: EventEmitter<void>;
+
+  /**
+   * Emitted when the link has been clicked.
+   */
+  @Event() gcdsClick!: EventEmitter<void>;
+
+  /*
+   * Observe lang attribute change
+   */
+  updateLang() {
+    const observer = new MutationObserver(mutations => {
+      if (mutations[0].oldValue != this.el.lang) {
+        this.lang = this.el.lang;
+      }
+    });
+    observer.observe(this.el, observerConfig);
+  }
+
+  componentWillLoad() {
+    // Validate attributes and set defaults
+    this.validateSize(this.size);
+
+    this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement, [
+      'referrerpolicy',
+    ]);
+
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+
+    this.updateLang();
+  }
+
+  render() {
+    const {
+      size,
+      lang,
+      display,
+      href,
+      rel,
+      target,
+      external,
+      download,
+      type,
+      inheritedAttributes,
+    } = this;
+
+    const attrs = {
+      href,
+      rel,
+      target,
+      download,
+      type,
+    };
+
+    const isExternal = target === '_blank' || external;
+
+    return (
+      <Host>
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions,jsx-a11y/click-events-have-key-events */}
+        <a
+          role="link"
+          tabIndex={0}
+          {...attrs}
+          class={`link--${size} ${display != 'block' ? `d-${display}` : ''}`}
+          ref={element => (this.shadowElement = element as HTMLElement)}
+          target={isExternal ? '_blank' : target}
+          rel={isExternal ? 'noopener noreferrer' : rel}
+          {...inheritedAttributes}
+          part="link"
+          onBlur={() => this.gcdsBlur.emit()}
+          onFocus={() => this.gcdsFocus.emit()}
+          onClick={() => this.gcdsClick.emit()}
+        >
+          <slot></slot>
+          {target === '_blank' || external ? (
+            <gcds-icon
+              name="external-link"
+              label={i18n[lang].external}
+              margin-left="200"
+            />
+          ) : download !== undefined ? (
+            <gcds-icon
+              name="download"
+              label={i18n[lang].download}
+              margin-left="100"
+            />
+          ) : href.toLowerCase().startsWith('mailto:') ? (
+            <gcds-icon
+              iconStyle="regular"
```

Comment:
Can you switch this prop to use kebab case please to stay consistent aka `icon-style`?